client/README.md

```diff
@@ -1,208 +1,389 @@
-# Python Exemplary Client
-
-This is an exemplary command-line tool to simplify the process of running a
-Batch SQL Translation job using the Google Cloud Bigquery Migration Python
-client package.
-
-## Supported Python Versions 
-Python >= 3.7
-
-## Installation
-
-Clone or download this repository and install the exemplary Python client: 
-
+# BigQuery Migration Service Batch SQL Translator CLI
+
+The [BigQuery Migration Service][BQMS] includes a product known as the
+[batch SQL translator][batch SQL translator] that supports translation of a 
+wide variety of SQL dialects to Google Standard SQL. This command line tool
+simplifies the process of using the batch SQL translator by providing a
+framework for pre and postprocessing of SQL files as well as upload/download of
+SQL files to/from GCS.
+
+- [Quickstart](#quickstart)
+- [Installation](#installation)
+- [Basic Usage](#basic-usage)
+  - [Environment Variables](#environment-variables)
+  - [config.yaml](#configyaml)
+  - [Recommended Usage](#recommended-usage)
+- [Metadata Lookup](#metadata-lookup)
+    - [DDL and Metadata Dump](#ddl-and-metadata-dump)
+    - [Unqualified References](#unqualified-references)
+- [Pre and Postprocessing](#pre-and-postprocessing)
+  - [Handling Macro/Templating Languages](#handling-macrotemplating-languages)
+  - [Custom Pre and Postprocessing Logic](#custom-pre-and-postprocessing-logic)
+- [Renaming SQL Identifiers](#renaming-sql-identifiers)
+- [Deploying to Cloud Run](#deploying-to-cloud-run)
+
+## Quickstart
+
+```shell
+# Install the CLI as described in the Installation section below.
+
+# Copy the example project directory to a project directory of your own.
+cp -R examples/teradata/sql <YOUR_PROJECT_DIRECTORY>
+
+# Change directory to your project directory.
+cd <YOUR_PROJECT_DIRECTORY>
+
+# Remove the example input files from the input directory.
```

Comment:
These two steps seem optional to me.

Reply:
Yes technically the customer can just copy their files into example directory in the source tree but since we are distributing via `git` that could get messy. I think we should encourage them to create a project directory outside the source tree.